spring-boot-project/spring-boot-tools/spring-boot-maven-plugin/src/main/java/org/springframework/boot/maven/AbstractRunMojo.java

```diff
@@ -169,6 +173,22 @@
 	@Parameter(property = "spring-boot.run.skip", defaultValue = "false")
 	private boolean skip;
 
+	/**
+	 * List of JVM options.
+	 * @since 2.0
+	 */
+	@Parameter(property = "spring-boot.run.jvmOptions")
+	private List<String> jvmOptions;
```

Comment:
I am not entirely convinced that this is required. This looks quite verbose with not a lot of benefit. Surefire has an [argLine parameter](http://maven.apache.org/surefire/maven-surefire-plugin/test-mojo.html#argLine) that matches what `jvmArguments` does so I'd keep that (no need for deprecation).

Reply:
ok